FrontEndReact/src/View/Admin/Add/AddTask/AdminAddAssessmentTask.js

```diff
@@ -121,6 +125,7 @@ class AdminAddAssessmentTask extends Component {
     }
     render() {
         var role_options = [];
+        var timezone_options = ['EST', 'CST', 'MST', 'PST'];
```

Comment:
Your timezone_options array should be an array of option tags with the value of each corresponding timezone and not just an array of the timezone strings. Your timezone_options array should be:
`var timezone_options = [`
`    <option value={"EST"} key={0}/>,`
`    <option value={"CST"} key={1}/>,`
`    <option value={"MST"} key={2}/>,`
`    <option value={"PST"} key={3}/>`
`];`

Reply:
Fixed!